plugins/scaffolder-actions/scaffolder-backend-module-utils/src/actions/json/merge.ts

```diff
@@ -77,3 +79,73 @@ export function createMergeJSONAction({ actionId }: { actionId?: string }) {
     },
   });
 }
+
+export function createMergeAction() {
+  return createTemplateAction<{ path: string; content: any }>({
+    id: 'roadiehq:utils:merge',
```

Comment:
Im not 100% convinced on the naming of these actions but you are just following the pattern to its good.

Reply:
Yeah neither am I. I considered breaking with the pattern but thought better of it.